packages/cli/test/unit/commands/project.test.ts

```diff
@@ -36,7 +34,33 @@ describe('project', () => {
         'Latest Production URL',
         'Updated',
       ]);
-      expect(data).toEqual([project.project.name, `--`]);
+      expect(data).toEqual([project.project.name, 'https://foobar.com']);
+    });
+    it('should list projects when there is no production deployment', async () => {
+      const user = useUser();
+      useTeams('team_dummy');
+      defaultProject.alias = [];
+      const project = useProject({
+        ...defaultProject,
+      });
+      console.log(project);
```

Comment:
**suggestion (soft-blocking):** remove line

```suggestion
```

In this or a follow-up PR: Can we remove this left over `console.log`?

Reply:
Thanks for catching!